test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -3,64 +3,120 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
 import org.junit.Test;
 
 public class UtilsTest {
     @Test
-    public void isAnyNull() {
+    public void emptyListShouldReturnFalse() {
```

Comment:
recollect that the test name should have 3 parts; first one being what is the method that is tested,